docs/hoon/library/1b.md

```diff
@@ -42,24 +50,69 @@ Examples
 
     > (cap 4)
     %2
+
     > (cap 6)
     %3
+
     > (cap (add 10 9))
     %2
 
+    > (cap 1)                                    ::address '1' is in neither the head nor the tail
+    ! exit
+
+    > (cap 0x40))                                
+    %2
+    > `@`0x40
+    64                           
+
+    > (cap 'a')                                  
+    %3
+    > `@`'a'
+    97                                        
 
 
 ***
 ### `++mas`
 
-Axis within head/tail?
-
-Computes the axis of `a` within either the head or tail of a noun (depends whether `a` lies within the the head or tail).
+Address within head/tail
+
+Computes the tree address of atom `a` within either the head (`+2`) or tail
+(`+3`) of a noun. In other words, `++mas` returns the relative address of `a`
+with respect to the head or the tail subtrees, depending on which one `a`
+descends from.
+
+            1
+          /   \
+         /     \
+        2       3              <--here are the head (`+2`) and the tail (`+3`)     
+       / \      /\
+      4   5    6  7
+     /\   /\  /\  /\
+     (continues...)
+
+Running `(mas 7)` in the Dojo will return `3`, because address `+3` is what
+`+7` now occupies. The tree below helps illustrate the relationship. With
+parentheses are `a` values (if `a` is in subtree `+3`), and without parentheses
+are the values returned with `(mas a)`.
+
+            1(3)                       ::new/(old) addresses
+           /    \
+          2       3
+         (6)     (7)
+        / \       /\
+       /   \     /  \
+      4     5   6    7
+     (12) (13) (14) (15)
+    / \    / \ / \   / \
+       (continues...)
+
+Notice how the old values in the head (subtree `+2`) were not illustrated in
+this case, because `+7` is within the tail (subtree `+3`).
 
```

Comment:
This is actually neat and it does add clarity.  Putting all this explanation above the "Accepts" and "Produces" sections seems a little strange though.  If you want to do these expanded commentaries, why not add an option "Explanation" section below, and leave the top for a brief, rigorous definition?